ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetBlockProcessor.java

```diff
@@ -53,7 +54,9 @@ boolean rewardCoinbase(
     if (skipZeroBlockRewards && blockReward.isZero()) {
       return true;
     }
-    final Wei coinbaseReward = blockReward.plus(blockReward.times(ommers.size()).dividedBy(32));
+    final UInt256 reward = UInt256.fromBytes(blockReward);
```

Comment:
Can Wei extend UInt256 or some delegating form of it?

Reply:
yes, BaseUInt256Value.

Reply:
That's all done now.